apps/chat/src/components/Sidebar/Sidebar.tsx

```diff
@@ -86,6 +87,10 @@ const Sidebar = <T,>({
 
   const isOverlay = useAppSelector(SettingsSelectors.selectIsOverlay);
 
+  const [windowWidth, setWindowWidth] = useState(
+    typeof window !== 'undefined' && window.innerWidth,
+  );
```

Comment:
here type will be `useState<number | false>` is it expected?

maybe it's better to write like `typeof window !== 'undefined' ? window.innerWidth : undefined`